### DISCUSSION TITLE
No format option for removable devices

### DISCUSSION THREAD

**Discussion by 4NXIE7Y** [moudey/Shell#73] [Q&A]
I can't seem to find the option to format removable storage devices...
![image](https://user-images.githubusercontent.com/98228741/205384463-09fc8dcc-cfad-478d-a9a7-856b92c99aee.png)

**Comment by moudey** [comment]
Press `Shift` + `Right-Click` and it will appear

**Comment by Natejoestev** [comment]
is there a way to make it appear in the non-shift click menu?
(what code does it take to add that option?)

**Reply by moudey** [reply]
This is in version 1.7 and has been moved to `more options` menu in 1.8
